spec/controllers/visits_controller_spec.rb

```diff
@@ -19,8 +21,6 @@
       end
     end
     context 'when user_id filter is sent' do
```

Comment:
Qué pasa cuando no se envía el user_id? Qué pasa si se envía un user_id que no existe? Deberíamos tener ambos casos de prueba!

Reply:
Agregemos un contexto que sea `when user_id filter is not sent`... debería retornar `bad_request`

Reply:
es para el caso de todas las visitas sin filtrar